client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -139,6 +142,13 @@ const Header: React.FunctionComponent = () => {
 	const hasPlaceholder =
 		!! siteTitle && ! recommendedDomainSuggestion && previousRecommendedDomain !== '';
 
+	const onDomainSelect = ( suggestion: DomainSuggestion | undefined ) => {
+		trackEventWithFlow( 'calypso_newsite_select_domain', {
```

Comment:
I think this needs to be `calypso_newsite_domain_select` so that _action_ is last.

`<source>_<context>_<optional subcontext>_<action>_<optional qualifier>`

See naming conventions document: PCYsg-4S2-p2